assets/dev/js/editor/controls/base-data.js

```diff
@@ -205,7 +205,9 @@ ControlBaseDataView = ControlBaseView.extend( {
 		try {
 			return parent && this.container.panel.getControlView( parent );
 			// eslint-disable-next-line no-empty
-		} catch ( e ) {}
+		} catch ( e ) {
+			return null;
```

Comment:
Notice that you are changing the behavior here.
Why not just this:
```suggestion
			return;
```

Reply:
it throws an error
will double check